cvat-canvas/src/typescript/canvasView.ts

```diff
@@ -1575,12 +1594,14 @@ export class CanvasViewImpl implements CanvasView, Listener {
     }
 
     private addPoints(points: string, state: any): SVG.PolyLine {
+        state.annotation_type = 'Manual';
```

Comment:
Why points are always ``manual``?

Reply:
From my understanding, TF auto annotation API does not support shapes other than rectangles.

Reply:
@Priya4607 
I believe if we introduce functionality that affects the whole CVAT, we shouldn't think only about TF Auto Annotation. There are (and will be) other methods to create automatic annotations. If it works only with tf annotation it should be implemented inside TF Auto Annotation plugin IMHO.